oaknational/oak-ai-lesson-assistant
# chore: maths rec-sys feature flag, fixes, and images in exports [AI-682]

## Description

- includes commits in https://github.com/oaknational/oak-ai-lesson-assistant/pull/357
  - i've temporarily disabled the export images part, as I need to iterate on it so that it fully supports our use case
- adds feature flag/ puts this functionality behind it
- fixes issue where `fetchExperimentalPatches` was checking LLM patches from every message, not just the latest patches


## To test

Add yourself to the feature flag groups:
- staging https://eu.posthog.com/project/8837/feature_flags/42665
- production https://eu.posthog.com/project/8212/feature_flags/42666

Then generate a Lesson Plan with subject Maths. You should see the starter and exit quizzes replaced with the placeholder maths quiz fixture.

If you're not in the feature flag group, you should see no change
